build/.eslintrc

```diff
@@ -0,0 +1,63 @@
+{
```

Comment:
Bower is not used in this project, rather npm. Yes it does match the Google style guide, taken from the samples [repo](https://github.com/webrtc/samples/blob/gh-pages/.eslintrc).